x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -241,7 +242,7 @@ const tooltipContent = {
   }),
   disabled: i18n.translate('xpack.lens.configPanel.color.tooltip.disabled', {
     defaultMessage:
-      'Individual series cannot be custom colored when the layer includes a “Break down by“',
+      'Individual series cannot be custom colored when the layer includes a “Break down by.“',
```

Comment:
Shouldn't this period be outside of the user-visible quotes?

Reply:
Nope, the official way you're supposed to deal with [punctuation at the end of a quote](https://www.grammarly.com/blog/quotation-marks/#:~:text=Quotation%20marks%20and%20other%20punctuation%20marks&text=In%20the%20United%20States%2C%20the,last%20night%2C%E2%80%9D%20Paul%20said.) is to keep it inside the quotation marks.